package.json

```diff
@@ -7,11 +7,15 @@
     "@material-ui/core": "3.5.1",
     "@material-ui/icons": "3.0.1",
     "@material-ui/lab": "3.0.0-alpha.23",
+    "firebase": "^5.5.9",
     "prop-types": "15.6.2",
     "react": "16.6.3",
     "react-dom": "16.6.3",
+    "react-redux": "^5.1.1",
     "react-router-dom": "^4.3.1",
     "react-scripts": "2.1.1",
+    "redux": "^4.0.1",
+    "redux-thunk": "^2.3.0",
     "styled-components": "4.1.1"
```

Comment:
We don't need it anymore. Please remove.
Also, I've noticed that you connect Roboto font in your index.html. Please remove it and use 
"typeface-roboto": "0.0.54" package instead.